bears/general/LineLengthBear.py

```diff
@@ -30,7 +46,11 @@ def run(self,
         :param indent_size:         Number of spaces per indentation level.
         :param ignore_length_regex: Lines matching each of the regular
                                     expressions in this list will be ignored.
+        :param language:            Programming language of the source code.
```

Comment:
should have also been moved up

Reply:
Yeah, forgot, will move up once core PR is merged so that there's only one next push :)